internal/a8n/types.go

```diff
@@ -189,6 +189,11 @@ func (c *ChangesetJob) Clone() *ChangesetJob {
 	return &cc
 }
 
+// Completed returns true for jobs that have already succesfully run
+func (c *ChangesetJob) Completed() bool {
```

Comment:
```suggestion
func (c *ChangesetJob) SuccessfullyCompleted() bool {
```

I'd rename this to keep it in sync with the behavior we have in `GetCampaignStatus`:

https://github.com/sourcegraph/sourcegraph/blob/19db137bc71f5d9e547e8c51205f2593456ffc77/enterprise/internal/a8n/store.go#L1435-L1448

There we count `ChangesetJob`s a `completed` even if they have an error.